### DISCUSSION TITLE
Counting Steps (Pedometer)

### DISCUSSION THREAD

**Discussion by osyyyS** [shinyorg/shiny#1516] [Q&A]
Hi, I'm trying to implement a MAUI App for IOS and Android which counts the user's daily steps (Pedometer). I have seen the [Plugin.Maui.Pedometer](https://github.com/jfversluis/Plugin.Maui.Pedometer) package, but I think it doesn't allow me to track the steps when the User isn't actively using the App. Could Shiny help me to always track the steps, or does anyone have any advice on how to do so?
I'm sorry if this is not the right place to ask this question 😅.

**Comment by aritchie** [comment]
This is the right place to ask, but Shiny can't make any process available in the background - it is specific to the processes we support.  For truly tracking steps, you really need the Apple Health/Google Fit

**Reply by osyyyS** [reply]
Thanks for the quick response, I just discovered your [Health](https://github.com/shinyorg/Health) repo. Can I use it for that?
